geomesa-stream/geomesa-stream-datastore/src/main/scala/org/locationtech/geomesa/stream/datastore/StreamDataStore.scala

```diff
@@ -46,27 +54,27 @@ case class FeatureHolder(sf: SimpleFeature, env: Envelope) {
   }
 }
 
-class StreamDataStore(source: SimpleFeatureStreamSource, timeout: Int) extends ContentDataStore {
+class StreamDataStore(source: SimpleFeatureStreamSource, timeout: Int, ns: Option[String]) extends ContentDataStore {
```

Comment:
i think preferably timeout would be either: 
1. defined as long as millis 
2. require a timeunit
3. be a string and use the time unit parsing code (e.g. "3 min", "6 hours")

Reply:
I'll add one

Reply:
added as a TODO: https://geomesa.atlassian.net/browse/GEOMESA-2161